README.md

```diff
@@ -5,6 +5,7 @@
 [![Build Status](https://travis-ci.org/pureconfig/pureconfig.svg?branch=master)](https://travis-ci.org/pureconfig/pureconfig)
 [![Coverage Status](https://coveralls.io/repos/github/pureconfig/pureconfig/badge.svg?branch=master)](https://coveralls.io/github/pureconfig/pureconfig?branch=master)
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.github.pureconfig/pureconfig_2.12/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.github.pureconfig/pureconfig_2.12)
+[![Javadocs](http://javadoc.io/badge/com.github.pureconfig/pureconfig_2.12.svg)](http://javadoc.io/doc/com.github.pureconfig/pureconfig_2.12)
```

Comment:
Thanks for doing this! I've wanted an easy to find link to these for a long time.
Suggestions: I'd expect the link to be called "Scaladocs or "Scala API," not "Javadocs." The [Spark approach](https://spark.apache.org/docs/2.2.0/api.html) has pretty good SEO. Alternatively [Scodec](http://scodec.org/api/) and http4s(https://http4s.org/v0.19/) call it simply "Scaladoc."
- Link to the [docs for `pureconfig` package](https://static.javadoc.io/com.github.pureconfig/pureconfig_2.12/0.9.1/pureconfig/index.html) directly instead of to the root, which is confusing because it's an empty page.

Reply:
Thanks for the suggestions, @leifwickland! I opted to change the link to "Scaladoc" (unfortunately, the badge still says "javadoc", but for SEO purposes it's not what counts). The changes are at 7644f8d.